pootle/apps/pootle_translationproject/models.py

```diff
@@ -79,15 +111,15 @@ def scan_translation_projects(languages=None, projects=None):
             project.disabled = True
             project.save()
         else:
-            lang_query = Language.objects.exclude(
-                    id__in=project.translationproject_set.live() \
-                                  .values_list('language', flat=True)
-                )
+            lang_query = Language.objects.all()
+
             if languages:
                 lang_query = lang_query.filter(code__in=languages)
 
             for language in lang_query.iterator():
-                create_or_resurrect_translation_project(language, project)
+                logging.info(u"Add background job for (%s, %s) processing" %
```

Comment:
Don't use % in logging.info calls:

`logging.info("Something %s something else %s", one, two)`